### PR TITLE
Added new UI

### PR DESCRIPTION
**Pull request by prykon**
Added the UI that freshens up the CSS but still conserves the progression circle feature.

### CODE REVIEW

File: magic-links/24hour.js [hunk 1]
```diff
@@ -95,13 +95,13 @@ jQuery(document).ready(function($) {
...
     let week_day_names = days_for_locale(navigator.language, 'narrow')
     let headers = `
-      <div class="day-cell week-day">${week_day_names[0]}</div>
-      <div class="day-cell week-day">${week_day_names[1]}</div>
-      <div class="day-cell week-day">${week_day_names[2]}</div>
-      <div class="day-cell week-day">${week_day_names[3]}</div>
-      <div class="day-cell week-day">${week_day_names[4]}</div>
-      <div class="day-cell week-day">${week_day_names[5]}</div>
-      <div class="day-cell week-day">${week_day_names[6]}</div>
+      <div class="day-cell week-day" style="color:black;font-size:12px;font-weight:550;margin-bottom:5px;">S</div>
```
Comment (corsacca) [inline magic-links/24hour.js:98]: @prykon we don't want `${week_day_names[0]}` to be replaced

Comment (corsacca) [reply magic-links/24hour.js:98]: lets make this css a class too :)

Comment (prykon) [reply magic-links/24hour.js]: Done and done. The css class is declared in 24hour.php

**Comment by corsacca** [comment]
Thanks @prykon!

**Pull request merged**